### ISSUE TITLE
The deploy icon is tiny

### ISSUE THREAD

**Issue by ThomasJClark** [WPIRoboticsProjects/GRIP#256]
The deploy icon in the menu is tiny my HiDPI screen.  I'm not sure if we can get controlsfx glyphs to scale correctly, so it might be a good idea to just use a png icon like the rest of the menu items.

**Comment by JLLeitschuh** [comment]
Can you open an issue with controlsfx?

**Comment by ThomasJClark** [comment]
I don't think this is a bug in ControlsFX.  Normally in JavaFX graphics don't inherit CSS rules

**Comment by JLLeitschuh** [comment]
We can set the size here.
Lets just add something to the DPI file to do this automatically:
http://controlsfx.bitbucket.org/org/controlsfx/glyphfont/GlyphFont.html

**Comment by ThomasJClark** [comment]
Can you figure out how to do that in FXML?

**Comment by JLLeitschuh** [comment]
No, can you. I already have enough issues tagged with my name on them right now.

**Comment by ThomasJClark** [comment]
I think it should be an FRC logo rather than an upload logo.  Once NT is merged it (which adds an FRC logo to the resources) I'll change it to do that

**Comment by AustinShalit** [comment]
Is this issue related to the picture attached?
![untitled](https://cloud.githubusercontent.com/assets/3799260/12109086/818fae4a-b333-11e5-9e71-a776b29ced2f.jpg)

**Comment by ThomasJClark** [comment]
> Is this issue related to the picture attached?

I don't think so, but that's an interesting problem I haven't seen before.  Mind opening up a new issue?

**Comment by PaulaRudy** [comment]
@JLLeitschuh does #279 address this as well?

**Comment by ThomasJClark** [comment]
No, but #239 does because it just sets deploying to use the same icon as
NetworkTables
